spec/setting_spec.rb

```diff
@@ -2,74 +2,79 @@
 
 describe ActiveSetting::Setting, 'with types/casting' do
   it 'should handle basic types and casting' do
-    s = ActiveSetting::Setting.new(:data_type => :integer, :raw_value => '6')
-    s.value.should == 6
+    s = ActiveSetting::Setting.new(data_type: :integer, raw_value: '6')
+    s.value.should eq 6
 
-    s = ActiveSetting::Setting.new(:data_type => :decimal, :raw_value => '7.6')
-    s.value.should == 7.6
+    s = ActiveSetting::Setting.new(data_type: :decimal, raw_value: '7.6')
+    s.value.should eq 7.6
 
-    s = ActiveSetting::Setting.new(:data_type => :symbol, :raw_value => 'hello')
-    s.value.should == :hello
+    s = ActiveSetting::Setting.new(data_type: :symbol, raw_value: 'hello')
+    s.value.should eq :hello
 
-    s = ActiveSetting::Setting.new(:raw_value => 'hello')
-    s.value.should == 'hello'
+    s = ActiveSetting::Setting.new(raw_value: 'hello')
+    s.value.should eq 'hello'
   end
 
   it 'should handle boolean types' do
-    s = ActiveSetting::Setting.new(:data_type => :boolean, :raw_value => '0')
+    s = ActiveSetting::Setting.new(data_type: :boolean, raw_value: '0')
     s.value.should be false
 
-    s = ActiveSetting::Setting.new(:data_type => :boolean, :raw_value => 'false')
+    s = ActiveSetting::Setting.new(data_type: :boolean, raw_value: 'false')
     s.value.should be false
 
-    s = ActiveSetting::Setting.new(:data_type => :boolean, :raw_value => '1')
+    s = ActiveSetting::Setting.new(data_type: :boolean, raw_value: '1')
     s.value.should be true
 
-    s = ActiveSetting::Setting.new(:data_type => :boolean, :raw_value => 'true')
+    s = ActiveSetting::Setting.new(data_type: :boolean, raw_value: 'true')
     s.value.should be true
   end
 
   it 'should handle multi value types including subvalue types and handle spacing' do
-    s = ActiveSetting::Setting.new(:data_type => :csv, :subtype => :integer, :raw_value => '1,2,3')
-    s.value.should == [1, 2, 3]
+    s = ActiveSetting::Setting.new(data_type: :csv, subtype: :integer, raw_value: '1,2,3')
+    s.value.should eq [1, 2, 3]
 
-    s = ActiveSetting::Setting.new(:data_type => :csv, :subtype => :symbol, :raw_value => 'first, second')
-    s.value.should == [:first, :second]
+    s = ActiveSetting::Setting.new(data_type: :csv, subtype: :symbol, raw_value: 'first, second')
+    s.value.should eq [:first, :second]
 
-    s = ActiveSetting::Setting.new(:data_type => :hash, :raw_value => 'a:1 , b : 2')
-    s.value.should == {:a => '1', :b => '2'}
+    s = ActiveSetting::Setting.new(data_type: :hash, raw_value: 'a:1 , b : 2')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.